### ISSUE TITLE
Mega download isn't working anymore..

### ISSUE THREAD

**Issue by ImYuter** [LoukaMB/SynapseX#354]
we need  a new download link, I accidently uninstalled synapse and when I wanted to download SynapseX back, mega removed it.
![image](https://user-images.githubusercontent.com/70919597/92417120-f9d16880-f136-11ea-98f3-ed2137c9cd60.png)

**Comment by BigMych** [comment]
That what I did to mega suck from this download

**Comment by BigMych** [comment]
The file you are trying to download is no longer available.
This link is unavailable as the user’s account has been closed for gross violation of MEGA’s Terms of Service.

**Comment by BigMych** [comment]
need a new link cuz mega not wroking right now i did uninstall to

**Comment by thatboisus** [comment]
yeah the mega link wont work, now it directs me to a anon files site. idk if it's safe or not

**Comment by BigMych** [comment]
its safe

**Comment by BigMych** [comment]
trust me its safe

**Comment by thatboisus** [comment]
> trust me its safe

 r u sure? did u try it?

**Comment by BigMych** [comment]
yes

**Comment by BigMych** [comment]
i did just try its gonna make your mind blown away its gonna work it work for me
